colcon-build-and-test/action.yaml

```diff
@@ -71,17 +71,13 @@ runs:
     - name: Test
       run: |
         . /opt/ros/${{ inputs.rosdistro }}/setup.sh
-        colcon lcov-result --initial --packages-above ${{ inputs.target-packages }}
+        colcon lcov-result --initial --verbose --packages-above ${{ inputs.target-packages }}
         colcon test --event-handlers console_cohesion+ \
           --mixin coverage-pytest \
           --packages-above ${{ inputs.target-packages }} \
           --return-code-on-test-failure
-        if [ -n "$(find build/ -name coverage.info)" ]; then
```

Comment:
Could you paste the result of `tree build/` after running `colcon test`?

Reply:
This is the result of autoware_auto_common after running colcon test.
```
build/
├── autoware_auto_cmake
│   ├── ament_cmake_core
│   │   ├── autoware_auto_cmakeConfig.cmake
│   │   ├── autoware_auto_cmakeConfig-version.cmake
│   │   ├── package.cmake
│   │   └── stamps
│   │       ├── ament_cmake_export_dependencies-extras.cmake.stamp
│   │       ├── ament_prefix_path.sh.stamp
│   │       ├── autoware_auto_cmake-extras.cmake.stamp
│   │       ├── nameConfig.cmake.in.stamp
│   │       ├── nameConfig-version.cmake.in.stamp
│   │       ├── package_xml_2_cmake.py.stamp
│   │       ├── package.xml.stamp
│   │       ├── path.sh.stamp
│   │       └── templates_2_cmake.py.stamp
│   ├── ament_cmake_environment_hooks
│   │   ├── ament_prefix_path.dsv
│   │   ├── local_setup.bash
│   │   ├── local_setup.dsv
│   │   ├── local_setup.sh
│   │   ├── local_setup.zsh
│   │   ├── package.dsv
│   │   └── path.dsv
│   ├── ament_cmake_export_dependencies
│   │   └── ament_cmake_export_dependencies-extras.cmake
│   ├── ament_cmake_index
│   │   └── share
│   │       └── ament_index
│   │           └── resource_index
│   │               ├── package_run_dependencies
│   │               │   └── autoware_auto_cmake
│   │               ├── packages
│   │               │   └── autoware_auto_cmake
│   │               └── parent_prefix_path
│   │                   └── autoware_auto_cmake
│   ├── ament_cmake_package_templates
│   │   └── templates.cmake
│   ├── ament_cmake_uninstall_target
│   │   └── ament_cmake_uninstall_target.cmake
│   ├── ament_lint_cmake
│   ├── cmake_args.last
│   ├── CMakeCache.txt
│   ├── CMakeFiles
│   │   ├── 3.16.3
│   │   │   └── CMakeSystem.cmake
│   │   ├── autoware_auto_cmake_uninstall.dir
│   │   │   ├── build.make
│   │   │   ├── cmake_clean.cmake
│   │   │   ├── DependInfo.cmake
│   │   │   └── progress.make
│   │   ├── cmake.check_cache
│   │   ├── CMakeDirectoryInformation.cmake
│   │   ├── CMakeOutput.log
│   │   ├── CMakeRuleHashes.txt
│   │   ├── Makefile2
│   │   ├── Makefile.cmake
│   │   ├── progress.marks
│   │   ├── TargetDirectories.txt
│   │   └── uninstall.dir
│   │       ├── build.make
│   │       ├── cmake_clean.cmake
│   │       ├── DependInfo.cmake
│   │       └── progress.make
│   ├── cmake_install.cmake
│   ├── colcon_build.rc
│   ├── colcon_command_prefix_build.sh
│   ├── colcon_command_prefix_build.sh.env
│   ├── CTestConfiguration.ini
│   ├── CTestCustom.cmake
│   ├── CTestTestfile.cmake
│   ├── install_manifest.txt
│   └── Makefile
├── autoware_auto_common
│   ├── ament_cmake_core
│   │   ├── autoware_auto_commonConfig.cmake
│   │   ├── autoware_auto_commonConfig-version.cmake
│   │   ├── package.cmake
│   │   └── stamps
│   │       ├── ament_cmake_export_include_directories-extras.cmake.stamp
│   │       ├── ament_prefix_path.sh.stamp
│   │       ├── nameConfig.cmake.in.stamp
│   │       ├── nameConfig-version.cmake.in.stamp
│   │       ├── package_xml_2_cmake.py.stamp
│   │       ├── package.xml.stamp
│   │       ├── path.sh.stamp
│   │       └── templates_2_cmake.py.stamp
│   ├── ament_cmake_environment_hooks
│   │   ├── ament_prefix_path.dsv
│   │   ├── local_setup.bash
│   │   ├── local_setup.dsv
│   │   ├── local_setup.sh
│   │   ├── local_setup.zsh
│   │   ├── package.dsv
│   │   └── path.dsv
│   ├── ament_cmake_export_include_directories
│   │   └── ament_cmake_export_include_directories-extras.cmake
│   ├── ament_cmake_gtest
│   │   └── test_common_gtest.txt
│   ├── ament_cmake_index
│   │   └── share
│   │       └── ament_index
│   │           └── resource_index
│   │               ├── package_run_dependencies
│   │               │   └── autoware_auto_common
│   │               ├── packages
│   │               │   └── autoware_auto_common
│   │               └── parent_prefix_path
│   │                   └── autoware_auto_common
│   ├── ament_cmake_package_templates
│   │   └── templates.cmake
│   ├── ament_cmake_uninstall_target
│   │   └── ament_cmake_uninstall_target.cmake
│   ├── ament_copyright
│   │   └── copyright.txt
│   ├── ament_cppcheck
│   │   └── cppcheck.txt
│   ├── ament_cpplint
│   │   └── cpplint.txt
│   ├── ament_lint_cmake
│   │   └── lint_cmake.txt
│   ├── ament_uncrustify
│   │   └── uncrustify.txt
│   ├── ament_xmllint
│   │   └── xmllint.txt
│   ├── cmake_args.last
│   ├── CMakeCache.txt
│   ├── CMakeFiles
│   │   ├── 3.16.3
│   │   │   ├── CMakeCCompiler.cmake
│   │   │   ├── CMakeCXXCompiler.cmake
│   │   │   ├── CMakeDetermineCompilerABI_C.bin
│   │   │   ├── CMakeDetermineCompilerABI_CXX.bin
│   │   │   ├── CMakeSystem.cmake
│   │   │   ├── CompilerIdC
│   │   │   │   ├── a.out
│   │   │   │   ├── CMakeCCompilerId.c
│   │   │   │   └── tmp
│   │   │   └── CompilerIdCXX
│   │   │       ├── a.out
│   │   │       ├── CMakeCXXCompilerId.cpp
│   │   │       └── tmp
│   │   ├── autoware_auto_common_uninstall.dir
│   │   │   ├── build.make
│   │   │   ├── cmake_clean.cmake
│   │   │   ├── DependInfo.cmake
│   │   │   └── progress.make
│   │   ├── cmake.check_cache
│   │   ├── CMakeDirectoryInformation.cmake
│   │   ├── CMakeError.log
│   │   ├── CMakeOutput.log
│   │   ├── CMakeRuleHashes.txt
│   │   ├── CMakeTmp
│   │   ├── Makefile2
│   │   ├── Makefile.cmake
│   │   ├── progress.marks
│   │   ├── TargetDirectories.txt
│   │   ├── test_common_gtest.dir
│   │   │   ├── build.make
│   │   │   ├── cmake_clean.cmake
│   │   │   ├── CXX.includecache
│   │   │   ├── DependInfo.cmake
│   │   │   ├── depend.internal
│   │   │   ├── depend.make
│   │   │   ├── flags.make
│   │   │   ├── link.txt
│   │   │   ├── progress.make
│   │   │   └── test
│   │   │       ├── gtest_main.cpp.gcda
│   │   │       ├── gtest_main.cpp.gcno
│   │   │       ├── gtest_main.cpp.o
│   │   │       ├── test_angle_utils.cpp.gcda
│   │   │       ├── test_angle_utils.cpp.gcno
│   │   │       ├── test_angle_utils.cpp.o
│   │   │       ├── test_bool_comparisons.cpp.gcda
│   │   │       ├── test_bool_comparisons.cpp.gcno
│   │   │       ├── test_bool_comparisons.cpp.o
│   │   │       ├── test_byte_reader.cpp.gcda
│   │   │       ├── test_byte_reader.cpp.gcno
│   │   │       ├── test_byte_reader.cpp.o
│   │   │       ├── test_float_comparisons.cpp.gcda
│   │   │       ├── test_float_comparisons.cpp.gcno
│   │   │       ├── test_float_comparisons.cpp.o
│   │   │       ├── test_mahalanobis_distance.cpp.gcda
│   │   │       ├── test_mahalanobis_distance.cpp.gcno
│   │   │       ├── test_mahalanobis_distance.cpp.o
│   │   │       ├── test_message_field_adapters.cpp.gcda
│   │   │       ├── test_message_field_adapters.cpp.gcno
│   │   │       ├── test_message_field_adapters.cpp.o
│   │   │       ├── test_template_utils.cpp.gcda
│   │   │       ├── test_template_utils.cpp.gcno
│   │   │       ├── test_template_utils.cpp.o
│   │   │       ├── test_type_name.cpp.gcda
│   │   │       ├── test_type_name.cpp.gcno
│   │   │       ├── test_type_name.cpp.o
│   │   │       ├── test_type_traits.cpp.gcda
│   │   │       ├── test_type_traits.cpp.gcno
│   │   │       └── test_type_traits.cpp.o
│   │   └── uninstall.dir
│   │       ├── build.make
│   │       ├── cmake_clean.cmake
│   │       ├── DependInfo.cmake
│   │       └── progress.make
│   ├── cmake_install.cmake
│   ├── colcon_build.rc
│   ├── colcon_command_prefix_build.sh
│   ├── colcon_command_prefix_build.sh.env
│   ├── colcon_command_prefix_test.sh
│   ├── colcon_command_prefix_test.sh.env
│   ├── colcon_test.rc
│   ├── compile_commands.json
│   ├── coverage_base.info
│   ├── CTestConfiguration.ini
│   ├── CTestCustom.cmake
│   ├── CTestTestfile.cmake
│   ├── gtest
│   │   ├── CMakeFiles
│   │   │   ├── CMakeDirectoryInformation.cmake
│   │   │   ├── gtest.dir
│   │   │   │   ├── build.make
│   │   │   │   ├── cmake_clean.cmake
│   │   │   │   ├── cmake_clean_target.cmake
│   │   │   │   ├── CXX.includecache
│   │   │   │   ├── DependInfo.cmake
│   │   │   │   ├── depend.internal
│   │   │   │   ├── depend.make
│   │   │   │   ├── flags.make
│   │   │   │   ├── link.txt
│   │   │   │   ├── progress.make
│   │   │   │   └── src
│   │   │   │       ├── gtest-all.cc.gcda
│   │   │   │       ├── gtest-all.cc.gcno
│   │   │   │       └── gtest-all.cc.o
│   │   │   ├── gtest_main.dir
│   │   │   │   ├── build.make
│   │   │   │   ├── cmake_clean.cmake
│   │   │   │   ├── cmake_clean_target.cmake
│   │   │   │   ├── CXX.includecache
│   │   │   │   ├── DependInfo.cmake
│   │   │   │   ├── depend.internal
│   │   │   │   ├── depend.make
│   │   │   │   ├── flags.make
│   │   │   │   ├── link.txt
│   │   │   │   ├── progress.make
│   │   │   │   └── src
│   │   │   │       ├── gtest_main.cc.gcno
│   │   │   │       └── gtest_main.cc.o
│   │   │   └── progress.marks
│   │   ├── cmake_install.cmake
│   │   ├── CTestTestfile.cmake
│   │   ├── libgtest.a
│   │   ├── libgtest_main.a
│   │   └── Makefile
│   ├── install_manifest.txt
│   ├── Makefile
│   ├── test_common_gtest
│   ├── Testing
│   │   ├── 20220131-0645
│   │   │   └── Test.xml
│   │   ├── TAG
│   │   └── Temporary
│   │       ├── CTestCostData.txt
│   │       ├── LastTest_20220131-0645.log
│   │       └── LastTest.log
│   └── test_results
│       └── autoware_auto_common
│           ├── copyright.xunit.xml
│           ├── cppcheck.xunit.xml
│           ├── cpplint.xunit.xml
│           ├── lint_cmake.xunit.xml
│           ├── test_common_gtest.gtest.xml
│           ├── uncrustify.xunit.xml
│           └── xmllint.xunit.xml
├── COLCON_IGNORE
└── compile_commands.json

62 directories, 212 files

```

Reply:
Thank you!